### DISCUSSION TITLE
What could cause many errors in the registry?

### DISCUSSION THREAD

**Discussion by thomasd3** [containrrr/watchtower#1859] [General]
The watchtower logs look fine:

<img width="699" alt="image" src="https://github.com/containrrr/watchtower/assets/14313218/2472b458-eda4-4ae8-be9e-bb4eaa8771eb">

But the registry logs show errors at every request:

<img width="1256" alt="Screenshot 2023-11-25 at 20 15 41" src="https://github.com/containrrr/watchtower/assets/14313218/45b5eec5-2c30-4147-9eb3-b2881e7120b6">

Sorry for pictures, for some reason my terminal emulator (Warp) doesn't let me copy-paste from that session.

**Comment by piksel** [comment]
The normal authentication flow for docker registries actually start with a 401, where the client get a header telling what scope needs authenticated. 

Ref: https://distribution.github.io/distribution/spec/auth/token/

If you want more details about what happens during an update, enable --debug.
